vision/cloud-client/product_search/import_product_sets.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.

```

Comment:
The generated libraries  use 'llc'
```
# Copyright 2018 Google LLC
```

Reply:
Some other python samples use Inc.  Keeping these samples consistent for now.